packages/server/src/db/index.js

```diff
@@ -1066,6 +1066,7 @@ async function getClosestGrants({
         .whereIn('grants_interested.agency_id', agencies.map((a) => a.id))
         .andWhere('close_date', '>=', timestamp)
         .andWhere('interested_codes.status_code', '!=', 'Rejected')
+        .groupBy('grants.title', 'grants.close_date', 'grants.grant_id')
```

Comment:
This "GROUP BY" has always needed to be here. Without it, a grant will be returned twice if both an agency and its sub-agency are interested in the grant.